configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,40 @@
+{
+    "activeSlotsCoeff": 0.05,
+    "systemStart": "2017-09-23T09:44:51Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 764824073,
+    "networkId": "Mainnet",
+    "epochLength": 216000,
+    "securityParam": 2160,
+    "slotLength": 1,
+    "slotsPerKESPeriod": 216000,
+    "maxKESEvolutions": 60,
+    "protocolParams": {
+        "a0": 0,
+        "decentralisationParam": 1,
+        "eMax": 0,
+        "extraEntropy": {
+            "tag": "NeutralNonce"
```

Comment:
@dcoutts Is this even used?

Reply:
In theory it could be, in practice we'll never use it. It should at least be optional so we don't have to mention it.

Reply:
@JaredCorduan could we make this optional, filling in `NeutralNonce` as the default.